sdk/digitaltwins/Azure.DigitalTwins.Core/tests/AdtQueryBuilderTests/AdtQueryBuilderTests.cs

```diff
@@ -92,5 +92,86 @@ public void AdtQueryBuilder_Select_Override()
                 .Should()
                 .Be("SELECT TOP(3) Room, Temperature FROM DigitalTwins");
         }
+
+        [Test]
+        public void AdtQueryBuilder_Where_Comparison()
+        {
+            new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereComparison("Temperature", QueryComparisonOperator.GreaterOrEqual, "50")
+                .Build()
+                .Stringify()
+                .ToUpper()
+                .Should()
+                .Be("SELECT * FROM DIGITALTWINS WHERE TEMPERATURE >= 50");
```

Comment:
What happens when you are comparing strings? 
like :
`SELECT * FROM DIGITALTWINS WHERE GivenName = 'hospital_room'` or something like that.

Reply:
same question is valid for other data types.

Reply:
This caught a pretty decent size hole in the current implementation. Addressing now, thank you so much for pointing that out.